package.json

```diff
@@ -163,7 +163,7 @@
 		"dev": "npm run build:packages && concurrently \"wp-scripts start\" \"npm run dev:packages\"",
 		"dev:packages": "node ./bin/packages/watch.js",
 		"docs:build": "node docs/tool",
```

Comment:
There is already `docs:build` command which is integrated into `precommit` hook. It would be ideal to consolidate those two commands.

Reply:
Consolidate them in e172a59